sketch_oct17a.ino

```diff
@@ -2,10 +2,35 @@
 
 void setup() {
   // put your setup code here, to run once:
-
+  int motor1_pin = 7; //vertical 
+  int motor2_pin = 8; //horizontal
+  int claw_pin = 9;
+  int pos = 0;
+  SphericalGear sg(motor1_pin,motor2_pin);
+  Servo claw_servo;
+  claw_servo.attach(claw_pin); // initialize motor
+  //we want the arm to reach down, then grab with the claw, then pull back up
+  //we might want to go back down and release the claw as well
+  
+  // open claw
+  openclaw(claw_servo, 45);
+  // move arm down
+  move(0,0,30);
```

Comment:
the first two parameters of the move function are the change in the angle of the arm. making them 0, means that the motors will not move at all. Change these to the amount of degrees you want the motor to move.